ppml/trusted-python-toolkit/Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG BASE_IMAGE_NAME
```

Comment:
organize the code as below
ARG
ENV
ADD
RUN

Reply:
Fixed.